.github/workflows/pull_request.yaml

```
@@ -108,6 +108,7 @@ jobs:
           '1.6.6',
           '1.7.1',
           '1.8.4'
+          '1.9.1'
```

Tests are complaining about the missing comma

```suggestion
          '1.8.4',
          '1.9.1'
```